t2.lic

```diff
@@ -41,11 +41,15 @@ class T2
         end
 
         # Skip if the skill is still on cooldown between trainings
-        next unless has_cooldown?(trainable['skill']) && cooldown_expired?(trainable['skill'])
+        if has_cooldown?(trainable['skill'])
```

Comment:
Style-wise, I'd prefer to see the conditionals combined

```suggestion
        if has_cooldown?(trainable['skill']) && !cooldown_expired?(trainable['skill'])
            next
        end
```

Reply:
Or one-line them like this:

```suggestion
        next if has_cooldown?(trainable['skill']) && !cooldown_expired?(trainable['skill'])
```

Reply:
Resolved in https://github.com/rpherbig/dr-scripts/pull/5410/commits/bcaeb120cc6206bceda7642768380696de2a971a